piv/src/main/java/com/yubico/yubikit/piv/PivSession.java

```diff
@@ -1458,7 +1457,7 @@ private int getRetriesFromCode(int statusCode) {
         if (statusCode == SW.AUTH_METHOD_BLOCKED) {
             return 0;
         }
-        if (version.isAtLeast(1, 0, 0) && version.isLessThan(1, 0, 4)) {
+        if (version.isLessThan(1, 0, 4)) {
```

Comment:
Why is this logic changed, was the old behavior incorrect?

Reply:
This is not part of the PR, but was used during development.